helm/charts/dsg-runner-infra/templates/sensor.yaml

```diff
@@ -31,7 +31,7 @@ spec:
                     restartPolicy: Never
                     containers:
                       - name: dsg-container
-                        image: "407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-data-service-generator:sha-27613bb"
+                        image: "407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-data-service-generator:feat/poc-mongodb-plugin"
```

Comment:
It needs update before we can merge it.

Reply:
I think this one is for development purposes and should be reverted. Make sure when you merge this PR this value represents the last image sha of the dsg